discovery-provider/alembic/trigger_sql/handle_track.sql

```diff
@@ -14,10 +14,16 @@ begin
   if TG_OP = 'UPDATE' then
```

Comment:
when do we directly update a track record? do i need to be accounting for changes to `is_unlisted` in an update?

Reply:
we only update a track record when making it unavailable 

https://github.com/AudiusProject/audius-protocol/blob/f2d308e3d10d1e4478d2bf07961462cdf300962b/discovery-provider/src/tasks/update_track_is_available.py#L113-L118

Reply:
we only update a track record when making it unavailable 

https://github.com/AudiusProject/audius-protocol/blob/f2d308e3d10d1e4478d2bf07961462cdf300962b/discovery-provider/src/tasks/update_track_is_available.py#L113-L118

so i think it should only be delta -1